src/routes/index.js

```diff
@@ -12,14 +15,24 @@ dotenv.config();
 const router = express.Router();
 
 router.use(bodyParser.json());
-router.use(bodyParser.urlencoded({ extended: false }));
+router.use(bodyParser.urlencoded({ extended: true }));
 router.use(cors());
+router.use(session({
+  secret: process.env.SECRET_KEY,
+  resave: true,
+  cookie: {
+    maxAge: 3600000,
+  },
+  saveUninitialized: true
+}));
+router.use(passport.initialize());
 const apiVersion = process.env.API_VERSION;
 
 const baseUrl = `/api/${apiVersion}`;
 
 router.get('/', (req, res) => res.status(200).json({ status: 200, data: 'Welcome to Authors Haven.' }));
 router.use(baseUrl, api);
+router.use(oauth);
```

Comment:
In my opinion, I think we should have our `api base url` before all routes in our API

Reply:
Let me harmonize